guides/doc-Provisioning_Guide/topics/proc_building-a-discovery-image.adoc

```diff
@@ -0,0 +1,119 @@
+[[building-a-discovery-image]]
+= Building a Discovery Image
+
+The Discovery image is a minimal operating system based on {RHEL} that is PXE-booted on hosts to acquire initial hardware information and to check in to the Satellite Server. Discovered hosts keep running the Satellite Discovery image until they are rebooted into Anaconda, which then initiates the provisioning process. The `foreman-discovery-image` package contains this image. You must install the package on the {SmartProxy} that provides TFTP services.
+
+Use this procedure to build a {Project} discovery image or rebuild an image if you change configuration files.
+
+Do not use this procedure on your production {Project} or {SmartProxy}.
+
+.Prerequisites
+
+Install the `livecd-tools` package:
+ifeval::["{build}" == "foreman"]
+----
+# yum install livecd-tools
+----
+endif::[]
+ifeval::["{build}" == "satellite"]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages install livecd-tools
+----
+endif::[]
```

Comment:
This could be as simple as a variable:

```
# {foreman_maintain_or_yum} livecd-tools
```

Upstream: `:foreman_maintain_or_yum: foreman-maintain packages install`

Downstream: `:foreman_maintain_or_yum: yum install`

Reply:
I will make a variable for this as part of another PR as it applies in many places and will rewrite at once. Thanks